<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="video_player" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>384</width>
     <height>216</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="video_frame" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>384</width>
     <height>216</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="choose_video">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>choose vide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="capture">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="analysis">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>analysis</string>
   </property>
  </widget>
  <widget class="QWidget" name="result_widget" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>reslut:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
